.github/workflows/release.yaml

```diff
@@ -153,6 +153,56 @@ jobs:
 
       - name: Add app metadata to package (MacOS)
         if: ${{ matrix.platform == 'macos' }}
+        run: |
+          Compress-Archive -Path "${{ env.PACKAGE_NAME }}" -DestinationPath "../${{ env.PACKAGE }}.zip"
```

Comment:
Merge broke here somehow, this isn't correct :fearful:

Reply:
[`6f9a860` (#154)](https://github.com/TheBevyFlock/bevy_quickstart/pull/154/commits/6f9a860fcdb1e95922bcb6aed512d9572408fbd5)